CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## Next Release API changes.
```

Comment:
We've been using the header `## master / unreleased` in the node_exporter for changelog entries.

Then we add a tag/date as part of doing the next release.

Reply:
good idea, thanks, updated!